### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_Screenreader:Voiceover is not reading the content and focus goes to setting button but it does not read as setting button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52403]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : MACOS High Sierra 10.13.5 Beta (17F70a)
**Additional Details:**
ScreenReader,Voiceover,MAS1.3.1
**Repro Steps:**
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to tree view under the search box.
4. Check what Voiceover is reading.

**Actual:**
Voiceover reads unnecessary things when focus goes to setting button which is not written on the screen.

**Expected:**
Voiceover should read meaningful word when focus goes to setting button.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User will not able to know what is the purpose the button.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

Does this issue occur when all extensions are disabled?: Yes/No
**Attachment For Reference**
[[Accessibility]A11y_VSCode_ViewExtensions_ScreenReader_Voiceover3.pptx](https://github.com/Microsoft/vscode/files/2118213/Accessibility.A11y_VSCode_ViewExtensions_ScreenReader_Voiceover3.pptx)

**Comment by isidorn** [comment]
What is unexpected here?
What behavior would you expect?

**Comment by isidorn** [comment]
Actually duplicate of https://github.com/Microsoft/vscode/issues/52596
